pageserver/src/tenant/config.rs

```diff
@@ -334,6 +334,11 @@ pub struct TenantConf {
     #[serde(with = "humantime_serde")]
     pub evictions_low_residence_duration_metric_threshold: Duration,
     pub gc_feedback: bool,
+
+    /// If set, the period between uploads of a heatmap from attached tenants.  This
+    /// may be disabled if a Tenant will not have secondary locations: only secondary
+    /// locations will use the heatmap uploaded by attached locations.
+    pub heatmap_period: Option<Duration>,
```

Comment:
It kind of itches to use magic values, but equally one ends up with an Option<Option<>> for defaulting otherwise. 
 Changed this to use zero for unset in d0e65b3ce9629813347d8ead96dd2b43e3db64c1